pkgs/development/python-modules/submitit/default.nix

```diff
@@ -0,0 +1,55 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  cloudpickle,
+  typing-extensions,
+  pytestCheckHook,
+}:
+
+buildPythonPackage rec {
+  pname = "submitit";
+  version = "1.2.0";
```

Comment:
This release is three years old. Is there any reason not to use the latest 1.5.2?